services/graph.js

```diff
@@ -1,136 +1,227 @@
-import 'isomorphic-fetch';
-
-export default class GraphService {
-  constructor(options) {
-    this.opts = options;
-  }
-
-  /**
-   * Get all tags
-   */
-  getAllTags() {
-    return fetch(
-      `${process.env.WRI_API_URL}/graph/query/list-concepts?application=${process.env.APPLICATIONS}&env=${process.env.API_ENV}`,
-      { headers: { 'Upgrade-Insecure-Requests': 1 } }
-
-    )
-      .then(response => response.json())
-      .then(response => response.data);
-  }
-  /**
-   * Get inferred tags
-   */
-  getInferredTags(tags) {
-    return fetch(
-      `${process.env.WRI_API_URL}/graph/query/concepts-inferred?concepts=${tags}&application=${process.env.APPLICATIONS}&env=${process.env.API_ENV}`,
-      { headers: { 'Upgrade-Insecure-Requests': 1 } }
-
-    )
-      .then(response => response.json())
-      .then(response => response.data);
+import WRISerializer from 'wri-json-api-serializer';
+
+// utils
+import { WRIAPI } from 'utils/axios';
+import { logger } from 'utils/logs';
+
+/**
+ * Get all tags
+ */
+export const fetchAllTags = (params = {
```

Comment:
I don't think it's a good idea having default params, you can override them accidentally at any moment and you still need them.